### PR TITLE
Fix Teacher Connections' fixed width container problems

### PR DESCRIPTION
**Pull request by jacbn**

### CODE REVIEW

File: src/scss/common/my-account.scss [hunk 1]
```diff
@@ -148,16 +148,19 @@
     margin-top: 2px;
   }
 
+  .connections-fixed-length-container {
+    max-width: calc(100% - 91.8px);
+  }
+
```
Comment (sjd210) [inline src/scss/common/my-account.scss:154]: This is ideal spacing for the "Leave" button with a tooltip icon on Isaac, but on Ada the font size is different and we don't have this tooltip so it leaves an awkward gap at the end. We should probably do something along these lines:

```suggestion
  .isaac-connections-fixed-length-container {
    max-width: calc(100% - 91.8px);
  }
  
  .ada-connections-fixed-length-container {
    max-width: calc(100% - 60px);
  }
```

(with a corresponding `classNames` change in the .tsx)

Comment (sjd210) [reply src/scss/common/my-account.scss:154]: On a similar note now that we have it, I would also add `connections-fixed-length-container` to lines [214](https://github.com/isaacphysics/isaac-react-app/blob/b3bf58207a03f18bc5257459aeb014b68e8f458b/src/app/components/elements/panels/TeacherConnections.tsx#L214) and [268](https://github.com/isaacphysics/isaac-react-app/blob/b3bf58207a03f18bc5257459aeb014b68e8f458b/src/app/components/elements/panels/TeacherConnections.tsx#L268) (although for Ada, we will again need another separate adjustment for its size). It's a less common problem than a group with multiple teachers, but students with particularly long names cause similar problems currently.

(From my testing, adding it directly to the `<span>` class alongside `connections-fixed-length-text` does the job with the different structure of that component).

**Review by sjd210 — CHANGES_REQUESTED** [review]

File: src/scss/common/my-account.scss (hunk 1 above)
Comment (jacbn) [reply src/scss/common/my-account.scss]: We tend to do the CSS changes with different .scss files (in /phy and /cs), but the idea is the same. I also was a bit unsure putting `-container` and `-text` on the same component was the right thing to do (even if it worked!) as the `max-width` lines were intended to be nested together, so I've cleaned that up a bit by making an actual container component.

Comment (sjd210) [reply src/scss/common/my-account.scss]: Of course - I promise I did know this! Regardless, I prefer this restructure of the container and everything now works well.

File: src/scss/phy/my-account.scss [hunk 2]
```diff
@@ -18,3 +18,14 @@
     width: max-content;
   }
 }
+
```
Comment (sjd210) [inline src/scss/phy/my-account.scss:21]: Nothing up with what you've done, but if we're fixing alignments here anyway then the icon in the header box on Physics has been offset and annoying me, so we might as well get this in at the same time.

```suggestion

.connect-list {
  .connect-list-header {
    .icon-person-active {
      margin-top: 7px;
    }
  
    h4 {
      padding: 0rem 1rem;
    }
  }
}

```

**Review by sjd210 — APPROVED** [review]
Another vaguely-related change I noticed and would like while we're touching this code anyway, but as long as you're happy with that then this can just be merged.

**Pull request merged**